### ISSUE TITLE
Embed Cockroach U videos in docs

### ISSUE THREAD

**Issue by jseldess** [cockroachdb/docs#6641]
- [ ] Identify optimal pages per video.
- [ ] Establish best practice for where to embed on page and how/when to promote Cockroach U.

**Comment by SpicyThrow** [comment]
We should also:
- [ ] Track where videos are embedded, for ease of maintenance. 

Your list is correct. I think this would require:

1.  A writer or writers to review the video content and decide what would be appropriate for cross-posting
2. Work with myself to create a list, by video, of the best doc page for cross-posting
3. Decide, by page, where and how it would be appropriate to post the content
4. Implement the videos on the pages
5. Establish a wiki page to track the location of each video

**Comment by SpicyThrow** [comment]
Next steps: Crossman to put together a wiki page that lists each video and the docs pages the video is linking to.

**Comment by SpicyThrow** [comment]
Created this page for tracking: https://cockroachlabs.atlassian.net/wiki/x/jgDaEw

**Comment by jseldess** [comment]
@SpicyThrow, I dropped the ball here. Do you think we should move forward? cc @dweatherbee

**Comment by SpicyThrow** [comment]
Sorry for the slow response, I was on vacation. I think some of the content from Getting Started and the Python course would be good for this but needs to re-branded. I don't know if re-branding is on the table for "performance basics" @dweatherbee ?

**Comment by dweatherbee** [comment]
Yes to all eventually. Re-branding of these course videos are all on the table but as a Lamont side projects after completing the core videos. Core videos require substantial content or are new videos. The rebranding videos only require green to blue/purple  rebranding. I expect most of them will be done by mid-late April.
